tests/foreman/destructive/test_rename.py

```diff
@@ -63,19 +63,20 @@ def test_positive_rename_satellite(module_org, module_product, module_target_sat
     username = settings.server.admin_username
     password = settings.server.admin_password
     old_hostname = module_target_sat.execute('hostname').stdout.strip()
-    new_hostname = f'new-{old_hostname}'
+    old_shortname, old_domain = old_hostname.split(".", 1)
+    new_hostname = f'{old_shortname}-changed.{old_domain}'
     # create installation medium with hostname in path
     medium_path = f'http://{old_hostname}/testpath-{gen_string("alpha")}/os/'
     medium = module_target_sat.api.Media(organization=[module_org], path_=medium_path).create()
     repo = module_target_sat.api.Repository(product=module_product, name='testrepo').create()
     # create /etc/hosts entry to pass s-c-h validation
     sat_ip = module_target_sat.execute(
```

Comment:
This can now print multiple IPs (on dual stack or multiple interfaces) so if you want to put entries in `/etc/hosts` you need to loop over it.

Reply:
Not supporting dualstack here. Let me just add IPv6 support first

Reply:
Actually I got access to dualstack machine so I can test the code there.
Leaning towards adding dualstack